dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.persistence-3.0.feature

```diff
@@ -5,6 +5,6 @@ IBM-Process-Types: server, \
  client
 -features=io.openliberty.jakarta.persistence.base-3.0
 -bundles=com.ibm.websphere.appserver.thirdparty.eclipselink.3.0; apiJar=false; location:=dev/api/third-party/; mavenCoordinates="org.eclipse.persistence:eclipselink:3.0.0"
-kind=noship
-edition=full
+kind=beta
```

Comment:
I would recommend starting everything off as noship, letting it go through a few builds, maybe an SOE ... then switch to beta.

Reply:
Reverted back to noship/full